hangman.js

```diff
@@ -1 +1,40 @@
-console.log("welcome to hangman");
\ No newline at end of file
+console.log("welcome to hangman");
+
+var game = {
+  word: "",
+  letters: [],
+  playerScore: 0,
+  executionerScore: 0,
+
+
+  getWord: function() {
+    this.word = prompt("Please enter the word that the player will guess.");
+    this.letters = this.word.split("");
+    this.guessLetter();
+  },
```

Comment:
Would be helpful if the user could see which letters he/she has guessed correctly and their positions in the word. Have seen other students create a string or array of `-`s, replacing correctly guessed letters as the game progresses. Let me know if you'd like to discuss how to do that.